packages/@lightningjs/ui-components/src/components/Artwork/Artwork.test.js

```diff
@@ -306,7 +306,7 @@ describe('Artwork', () => {
     };
     await artwork.__updateSpyPromise;
     expect(artwork._FillColor).not.toBeUndefined();
-    expect(artwork._FillColor.color).toBe(0xfff663399);
+    expect(artwork._FillColor.color).toBe(3004700697); // Equal to 0xfff663399 decimal format
```

Comment:
does this change matter? I feel like the previous is easier to parse and know what color is being used

Reply:
With the new withThemeStyles process Javascript is interpreting the hexadecimal value as a decimal number. I am not 100% sure why this test had to change to pass to be honest.